roles/network_plugin/calico/tasks/check.yml

```diff
@@ -53,7 +53,7 @@
 - name: "Check if inventory match current cluster configuration"
   assert:
     that:
-      - calico_pool_conf.spec.blockSize|string == (calico_pool_blocksize | default(kube_network_node_prefix | string))
+      - calico_pool_conf.spec.blockSize|int == (calico_pool_blocksize | default(kube_network_node_prefix) | int)
```

Comment:
calico_pool_conf.spec.blockSize is an integer as the following, the change itself seems good for me.
```
# calicoctl.sh get ipPool default-pool -o json
{
...
  },
  "spec": {
    "cidr": "10.233.64.0/18",
    "vxlanMode": "Always",
    "ipipMode": "Never",
    "natOutgoing": true,
    "blockSize": 24,
    "nodeSelector": "all()"
  }
}
```